bin/ycsb.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/sh
```

Comment:
Not sure what you mean?  The shellcheck tool you refer to checks for "bashisms" if #!/bin/sh is specified.

That being said, I'm having some issue with the shellcheck tool.  A few of the recommend changes are causing the script to not find the jars I specify.